2021-04-06 15:25:44,621 INFO    MainThread:31166 [wandb_setup.py:_flush():69] setting env: {'project': 'genre_classification_prod', 'run_group': 'dev'}
2021-04-06 15:25:44,621 WARNING MainThread:31166 [wandb_setup.py:_flush():69] could not save program above cwd: /home/giacomov/miniconda3/envs/mlflow-7398531e5288219e428182dc4d6601ae57991d39/bin/pytest
2021-04-06 15:25:44,621 INFO    MainThread:31166 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-06 15:25:44,621 INFO    MainThread:31166 [wandb_init.py:_log_setup():334] Logging user logs to /home/giacomov/udacity/repo/nd0821-c2-build-model-workflow/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/solution/check_data/wandb/run-20210406_152544-3e70gxec/logs/debug.log
2021-04-06 15:25:44,622 INFO    MainThread:31166 [wandb_init.py:_log_setup():335] Logging internal logs to /home/giacomov/udacity/repo/nd0821-c2-build-model-workflow/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/solution/check_data/wandb/run-20210406_152544-3e70gxec/logs/debug-internal.log
2021-04-06 15:25:44,623 INFO    MainThread:31166 [wandb_init.py:init():367] calling init triggers
2021-04-06 15:25:44,623 INFO    MainThread:31166 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-04-06 15:25:44,624 INFO    MainThread:31166 [wandb_init.py:init():416] starting backend
2021-04-06 15:25:44,624 INFO    MainThread:31166 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-06 15:25:44,650 INFO    MainThread:31166 [backend.py:ensure_launched():123] starting backend process...
2021-04-06 15:25:44,655 INFO    MainThread:31166 [backend.py:ensure_launched():127] started backend process with pid: 31191
2021-04-06 15:25:44,656 INFO    MainThread:31166 [wandb_init.py:init():421] backend started and connected
2021-04-06 15:25:44,658 INFO    MainThread:31166 [wandb_init.py:init():461] updated telemetry
2021-04-06 15:25:44,659 INFO    MainThread:31166 [wandb_init.py:init():480] communicating current version
2021-04-06 15:25:45,110 INFO    MainThread:31166 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.10.25 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-06 15:25:45,110 INFO    MainThread:31166 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-04-06 15:25:45,311 INFO    MainThread:31166 [wandb_init.py:init():518] starting run threads in backend
2021-04-06 15:25:45,382 INFO    MainThread:31166 [wandb_run.py:_console_start():1476] atexit reg
2021-04-06 15:25:45,384 INFO    MainThread:31166 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-04-06 15:25:45,384 INFO    MainThread:31166 [wandb_run.py:_redirect():1344] Redirecting console.
2021-04-06 15:25:45,384 INFO    MainThread:31166 [redirect.py:install():213] install start
2021-04-06 15:25:45,385 INFO    MainThread:31166 [redirect.py:install():228] install stop
2021-04-06 15:25:45,385 INFO    MainThread:31166 [redirect.py:install():213] install start
2021-04-06 15:25:45,386 INFO    MainThread:31166 [redirect.py:install():228] install stop
2021-04-06 15:25:45,386 INFO    MainThread:31166 [wandb_run.py:_redirect():1390] Redirects installed.
2021-04-06 15:25:45,386 INFO    MainThread:31166 [wandb_init.py:init():542] run started, returning control to user process
2021-04-06 15:25:47,400 INFO    MainThread:31166 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-04-06 15:25:47,401 INFO    MainThread:31166 [wandb_run.py:_restore():1418] restore
2021-04-06 15:25:47,401 INFO    MainThread:31166 [redirect.py:uninstall():232] uninstall start
2021-04-06 15:25:47,401 INFO    MainThread:31166 [redirect.py:_stop():287] _stop: stdout
2021-04-06 15:25:47,401 INFO    MainThread:31166 [redirect.py:_stop():293] _stop closed: stdout
2021-04-06 15:25:47,401 INFO    stdout    :31166 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-04-06 15:25:47,401 INFO    stdout    :31166 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:_stop():299] _stop joined: stdout
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:_stop():301] _stop rd closed: stdout
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:uninstall():236] uninstall done
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:uninstall():232] uninstall start
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:_stop():287] _stop: stderr
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:_stop():293] _stop closed: stderr
2021-04-06 15:25:47,402 INFO    stderr    :31166 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-04-06 15:25:47,402 INFO    stderr    :31166 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:_stop():299] _stop joined: stderr
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:_stop():301] _stop rd closed: stderr
2021-04-06 15:25:47,402 INFO    MainThread:31166 [redirect.py:uninstall():236] uninstall done
2021-04-06 15:25:47,561 INFO    MainThread:31166 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1048
  total_bytes: 1048
}

2021-04-06 15:25:49,566 INFO    MainThread:31166 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3554
  total_bytes: 3554
}

2021-04-06 15:25:50,642 INFO    MainThread:31166 [wandb_run.py:_show_files():1790] logging synced files
